devsetup/tripleo/net_config.j2

```diff
@@ -14,7 +14,14 @@ network_config:
   - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_cidr }}
   - ip_netmask: {{ ctlplane_ip }}/32
   - ip_netmask: {{ ctlplane_vip }}/32
+  {% if manage_default_route|bool %}
```

Comment:
we should include a default false here in case this is not defined (we use this template to configure networking in ci without calling the tripleo.sh script)

Reply:
i have a similar concern about the default_route
I arrived here following the TRIPLEO_NETWORKING
The intention is that this will only affect the multicell scenario/job but the default given is TRUE 

```
EDPM_CONFIGURE_NETWORKING ?= true
```

We could either make the default be false, and as @cescgina pointed out default it here, or the inverse, which is what you proposed with https://review.rdoproject.org/r/c/rdo-jobs/+/54221 where you explicitly set it false for the vanila multinode case

```
            EDPM_CONFIGURE_NETWORKING: false  

```

In my opinion it is more appropriate for this to default to false and then you can explicitly set this in the multi-cell job where is will be needed.

In any case though and before we merge we must run the multinode jobs so we can sanity check they are ok with these changes. I know you've run them in the past but we should run them again once this is ready for merge  as there are many ongoing changes here.